build.gradle

```diff
@@ -41,7 +41,7 @@ ext {
     "org.codehaus.groovy:groovy-xml:${groovyVersion}",
   ]
   groovyConsoleExtraDependency = [
-    "org.codehaus.groovy:groovy-groovysh:${groovyVersion}"
+    "org.codehaus.groovy:groovy-console:${groovyVersion}"
```

Comment:
Would it work also for Groovy 2.5?

Reply:
Yes, it always needed only the `groovy-console` dependency which is a transitive dependency of `groovy-groovysh`.

Reply:
Great that you spot it!